dms-core/src/main/kotlin/team/aliens/dms/domain/manager/dto/ManagerResponse.kt

```diff
@@ -0,0 +1,26 @@
+package team.aliens.dms.domain.manager.dto
+
+import team.aliens.dms.domain.school.model.School
+import java.util.UUID
+
+data class ManagerDetailsResponse(
+    val schoolId: UUID,
+    val schoolName: String,
+    val code: String,
+    val question: String,
+    val answer: String
+) {
+    companion object {
+        fun of(school: School) = ManagerDetailsResponse(
+            schoolId = school.id,
+            schoolName = school.name,
+            code = school.code,
+            question = school.question,
+            answer = school.answer
+        )
```

Comment:
p2 도메인 하나 쓰는건 `run`으로 묶어주면 좋을 것 같아요
```suggestion
        fun of(school: School) = school.run {
            ManagerDetailsResponse(
                schoolId = school.id,
                schoolName = school.name,
                code = school.code,
                question = school.question,
                answer = school.answer
            )
        }
```

Reply:
run을 사용하면 school. 안하고 그냥 id 사용할수있지않나요?

Reply:
헉 실수였습니다